pkg/cmd/dockerregistry/dockerregistry.go

```diff
@@ -98,6 +98,38 @@ func Execute(configFile io.Reader) {
 		pruneAccessRecords,
 	)
 
+	// openshiftRouter groups the OpenShift related registry extensions
+	openshiftRouter := app.NewRoute().PathPrefix("/openshift/").Subrouter()
+
+	signatureRouter := openshiftRouter.NewRoute().PathPrefix("/signature/").Subrouter()
+	signatureAccessRecords := func(*http.Request) []auth.Access {
+		return []auth.Access{
+			{
+				Resource: auth.Resource{
+					Type: "signature",
+				},
+				Action: "get",
+			},
+		}
+	}
+
+	app.RegisterRoute(
+		// GET /openshift/signature/<reference>
+		// FIXME: Because the reference.ReferenceRegexp is anchored, we have to strip the
+		// leading '^' from the regexp in order to match the image reference in a sub-path.
+		signatureRouter.Path("/{reference:"+strings.TrimPrefix(reference.ReferenceRegexp.String(), "^")+"}").Methods("GET"),
```

Comment:
We either need an endpoint to write signatures to land _at the same time_, or we need an equivalent of #12480 (with the master API address) resurrected; otherwise skopeo would be writing signatures to a different place (“sigstore”) from which it would try reading (this new API).